beta/kustomize/README.md

```diff
@@ -0,0 +1,214 @@
+
+From the Root Folder, redeploy the bridge using the command listed in step 4. The
+SCIM bridge will restart using this new deployment, and configure the Let's Encrypt
+integration to issue and use a certificate.
+
+## Step 6: Connect your identity provider
+
+To finish setting up automated user provisioning, [connect your identity provider to the SCIM bridge](https://support.1password.com/scim/#step-3-connect-your-identity-provider).
+
+## Update your SCIM bridge
+
+👍 Check for 1Password SCIM Bridge updates on the [SCIM bridge release page](https://app-updates.agilebits.com/product_history/SCIM).
+
+To update the SCIM bridge, connect to your Kubernetes cluster and run the following command, replacing v2.8.0 with the latest version:
+
+```bash
+kubectl set image deploy/op-scim-bridge op-scim-bridge=1password/scim:v2.8.3
+```
+
+The update should take 2-3 minutes for Kubernetes to complete.
+
+## Appendix: Resource recommendations
+
+The default resource recommendations for the SCIM bridge and Redis deployments are acceptable in most scenarios, but they may fall short in high-volume deployments where a large number of users and/or groups are being managed. We strongly recommend increasing the resources for both the SCIM bridge and Redis deployments.
+
+| Expected Provisioned Users | Resources                   |
+| -------------------------- | --------------------------- |
+| 1-1000                     | Default                     |
+| 1000-5000                  | High Volume Deployment      |
+| 5000+                      | Very High Volume Deployment |
+
+Our current default resource requirements (defined in [op-scim-deployment](https://github.com/1Password/scim-examples/blob/master/kubernetes/op-scim-deployment.yaml#L29) and [redis-deployment.yaml](https://github.com/1Password/scim-examples/blob/master/kubernetes/redis-deployment.yaml#L21)) are:
+
+<details>
+  <summary>Default</summary>
+
+```yaml
+requests:
+  cpu: 125m
+  memory: 256M
+
+limits:
+  cpu: 250m
+  memory: 512M
+```
+
+</details>
+
+Note that these are the recommended `requests` and `limits` values for both the SCIM bridge and Redis containers. These values can be scaled down again to the default values after the initial large provisioning event.
+
+<details>
+  <summary>High Volume Deployment</summary>
+
+```yaml
+requests:
+  cpu: 500m
+  memory: 512M
+
+limits:
+  cpu: 1000m
+  memory: 1024M
+```
+
+</details>
+
+<details>
+  <summary>Very High Volume Deployment</summary>
+
+```yaml
+requests:
+  cpu: 1000m
+  memory: 1024M
+
+limits:
+  cpu: 2000m
+  memory: 2048M
+```
+
+</details>
+
+Configuring these values can be done with Kubernetes commands. You can get the names of the deployments with `kubectl get deployments`.
+
+```bash
+# scale down deployment
+kubectl scale --replicas=0 deployment/op-scim-bridge
+
+# scale down redis deployment
+kubectl scale --replicas=0 deployment/op-scim-bridge-redis-master
+
+# update op-scim-redis resources
```

Comment:
As a future todo for this deployment example, we might want to add overlays for the different resource recommendations.